### DISCUSSION TITLE
Adding new translation

### DISCUSSION THREAD

**Discussion by Nerlight** [Jorixon/JASM#128] [General]
I could translate the interface into Russian if you would like to accept my help.

**Comment by Jorixon** [comment]
Hello, sorry for late response I've been busy with life and baldurs gate 3...

I don't have much experience with localization, but I've laid some groundwork for localizing the app. 

If you're still interested in assisting with the Russian translation, we can start with startup page and then slowly work our way through the app. It'll take some time on my side as well as most text is hard coded into the UI, the same goes for notifications

**Reply by Nerlight** [reply]
My apologies, same here.

I'm ready to start at any time convenient for you.
I would like to know how can I provide assistance and speed up the work process for you. 
I don't have much experience in translating apps and I don't know what editing ways you use yet, so I'd be glad if you shared your method.
